src/mlpack/methods/rann/krann_main.cpp

```diff
@@ -150,7 +150,19 @@ static void mlpackMain()
   const int lsInt = CLI::GetParam<int>("leaf_size");
   RequireParamValue<int>("leaf_size", [](int x) { return x > 0; }, true,
       "leaf size must be greater than 0");
-
+    
+  // Sanity check on tau.
+  RequireParamValue<double>("tau", [](double x) { return (x >= 0.0 && x <=100.0);
+  		},
+      true, "tau must be in range [0.0, 100.0]");
+  const double tau = CLI::GetParam<double>("tau");
+    
+   // Sanity check on alpha.
+  RequireParamValue<double>("alpha", [](double x) { return (x >= 0.0 && x <=1.0);
+  		},
+      true, "alpha must be in range [0.0, 1.0]");
```

Comment:
The formatting seems strange on these lines; do you think that you could clean it up? :+1: